packages/webview_flutter/webview_flutter_wkwebview/ios/Classes/FWFWebViewHostApi.m

```diff
@@ -201,7 +201,9 @@ - (void)setInspectableForWebViewWithIdentifier:(NSNumber *)identifier
                                    inspectable:(NSNumber *)inspectable
                                          error:(FlutterError *_Nullable *_Nonnull)error {
   if (@available(macOS 13.3, iOS 16.4, tvOS 16.4, *)) {
+#if __MAC_OS_X_VERSION_MAX_ALLOWED >= 130300 || __IPHONE_OS_VERSION_MAX_ALLOWED >= 160400
```

Comment:
Hm, looking at this again: do `@available` checks evaluate to false automatically when the compile SDK is lower than the available version? I'm not sure they do, and if they don't then this would, if compiled with an SDK < 16.4, but run on 16.4, silently no-op.

I guess that's not that big a deal for this particular case, but we should investigate before using this pattern again.

Reply:
> I'm not sure they do, and if they don't then this would, if compiled with an SDK < 16.4, but run on 16.4, silently no-op.

Yea, I think you're correct. This could probably use an additional `#ifelse` that also returns a `FlutterError`. I didn't quite understand this until after I autosubmitted.